src/jit/lsraxarch.cpp

```diff
@@ -2959,9 +2965,9 @@ int LinearScan::BuildMul(GenTree* tree)
 //    isFloatingPointType   - true if it is floating point type
```

Comment:
nit: This comment should have been fixed up

Reply:
Will do in next PRs. Thanks.